workshops/scipy2023/README.md

```diff
@@ -19,6 +19,18 @@ See instructions to set up the environment for running the tutorial material [he
 For the live tutorial, we will be using the SciPy 2023 Nebari JupyterHub. Instructions for the live
 tutorial will be added soon.
 
+## Github Codespaces
+
+This tutorial is available to run within github codespaces with the conda environment specification
+in the [`conda-lock.yml`](../../conda/conda-lock.yml) file.
+
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new/xarray-contrib/xarray-tutorial/tree/main?devcontainer_path=.devcontainer%2Fscipy2023%2Fdevcontainer.json)
+
+☝️ Click the button above to launch the github codespaces.
+Once it's launched, a terminal will open and it will execute `jupyter lab` automatically.
+Once you see a url to click, simply `cmd + click` the given url.
+This will open up another tab in your browser, leading to a Jupyter Lab Interface.
```

Comment:
Format as itemized list? A little more helpful text about what Codespaces is (is it free?) would be great too.